lib/chainlock/chainlock.js

```diff
@@ -126,28 +126,68 @@ ChainLock.fromObject = function fromObject(obj) {
   const data = ChainLock._fromObject(obj);
   return new ChainLock(data);
 };
+
 /**
  * Verify that the signature is valid against the Quorum using quorumPublicKey
  * @param {QuorumEntry} quorumEntry - quorum entry to test signature against
- * @returns {Promise<Boolean>} - if the signature is valid for this quorum
```

Comment:
Is there still cases when we need it publicly?

Reply:
done